cli/src/cmd/mod.rs

```diff
@@ -92,9 +92,28 @@ If you are in a subdirectory in a Git repository, try adding `--root .`"#,
     } else if output.is_unchanged() {
         println!("no files changed, compilation skipped.");
     } else {
+        let mut contracts = BTreeMap::new();
+        for (_, name, _, version) in
+            output.clone().output().contracts.contracts_with_files_and_version()
+        {
+            contracts
+                .entry(version.to_string())
+                .or_insert(Vec::<String>::new())
+                .push(name.to_string());
+        }
+
+        println!("compiled contracts:");
+        for (key, value) in contracts.clone().into_iter() {
+            println!("compiler version: {}", key);
```

Comment:
the `Version` display implementation prints everything, instead we could do here `{}.{}.{}", version.major, version.minor, version.patch`

Reply:
Will update, thanks for the review